go/vt/vtctl/grpcvtctldclient/Makefile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 The Vitess Authors.
```

Comment:
Oh... right, it's 2021 now. 😭 Haha. I guess this applies to all the files in this branch.